tests/test_file_utils.py

```
@@ -0,0 +1,1078 @@
+        result = _is_builtin_custom_models_config(None)
+        assert result == False
+
+
+class TestMCPDirectory:
+    """Test cases for is_mcp_directory function"""
+
+    def test_is_mcp_directory_false_for_non_mcp(self):
+        """Test non-MCP directory returns False"""
+        with tempfile.TemporaryDirectory() as temp_dir:
+            result = is_mcp_directory(Path(temp_dir))
+            assert result == False
```

Ruff (0.11.9)

Avoid equality comparisons to `False`; use `if not result:` for false checks

Replace with `not result`

(E712)